demo/pages/checkable-rows/index.html

```diff
@@ -0,0 +1,30 @@
+<q-grid columns="$ctrl.columns"
+		  rows="$ctrl.rows"
+		  selection="$ctrl.selection"
+		  selection-mode="$ctrl.selectionMode"
+		  on:selection-changed="$ctrl.selectionChanged($event)">
+	<q-grid:check>
```

Comment:
need to under, what is beeter select-all vs head select vs body select-all vs check-all vs checkbox-all vs box-all